mtrf/matrices.py

```diff
@@ -1,6 +1,19 @@
 import numpy as np
 
 
+def _reduced_covariance_matrices(xs, ys, lags, zeropad=True, bias=True):
```

Comment:
Do we need an extra function for this? We could also add a boolean argument `average` to the covariance_matrices function

Reply:
I feel adding too much functionality into the covariance_matrices will make it too complicated (additional if..else.. branch). And because we need to accumulatively sum the cov_mat for each trial (to save memory), we still need to add several lines of code into the covariance_matrices, given that currently we use a list to store the cov_mat for each trial.